### ISSUE TITLE
Creating a task with dueDate before form submission

### ISSUE THREAD

**Issue by Dinith1** [se701g2/Doto#200]
### Bug Summary: 
- When you create a new task with a dueDate before the time you submit the modal form it causes the task to be scheduled in parallel to another task.

--- 

### Steps To Reproduce

1) Create a task with the dueDate set to any time before clicking the submit/create button
2) New task overlaps with an existing task when scheduled

### Expected behaviour

- This is to be discussed
  - Either don't allow the task to be made
  - Or indicate to the user that this action is invalid
  - Or some other appropriate action

### Observed Behaviour

- New task overlaps with an existing task when scheduled

### Screenshots

![image](https://user-images.githubusercontent.com/34013038/77130973-bf0f4180-6abe-11ea-9e13-a988c6db4936.png)
